### PR TITLE
fix: change UI of describe picture page

### PR DESCRIPTION
**Pull request by DanHuy77**

### CODE REVIEW

File: the-telephone-app-fe/src/components/Draw/Draw.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 .draw-header {
     background-color: var(--main-color);
+    box-shadow: 1px 3px 4px 2px #888888
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/Draw/Draw.css]: add semicolon

File: the-telephone-app-fe/src/components/Draw/Draw.css [hunk 2]
```diff
@@ -21,6 +22,7 @@
     height: 48px;
     font-family: Fredoka;
     margin-left: 2rem;
+    box-shadow: 1px 3px 4px 2px #888888
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/Draw/Draw.css]: add semicolon

File: the-telephone-app-fe/src/components/Draw/Draw.css [hunk 3]
```diff
@@ -45,6 +47,7 @@
 .draw-paper {
     border: 4px solid var(--main-color);
     border-radius: 4px;
+    box-shadow: 1px 3px 4px 2px #888888
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/Draw/Draw.css]: add semicolon

File: the-telephone-app-fe/src/components/DescribePicture/DescribePicture.js [hunk 4]
```diff
@@ -1,18 +1,20 @@
...
 import "./DescribePicture.css";
-import imgLogo from "../../assets/gartic.svg";
+import imgLogo from "../../assets/header.png";
 import { useLocation } from "react-router-dom";
 import React, { useState, useEffect, useRef } from "react";
 import axios from "axios";
+import { BsFillCheckCircleFill } from "react-icons/bs";
 
-const DescribePicture = () => {
+const Temp = () => {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/DescribePicture/DescribePicture.js]: Why it is Temp?

File: the-telephone-app-fe/src/components/DescribePicture/DescribePicture.css [hunk 5]
```diff
@@ -1,130 +1,20 @@
-body {
-    /* background-color: #6f90b3; */
+.drew-picture {
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/DescribePicture/DescribePicture.css]: Should be drawn-picture

**Review by thoa01 — APPROVED** [review]

File: the-telephone-app-fe/src/assets/index.css [hunk 6]
```diff
@@ -2,4 +2,5 @@
     --bg-color: #FFF9EB;
     --main-color: #FABA9F;
     --secondary-color: #C3C9E8;
+    --block-box-shadow: 1px 3px 4px 2px #888888
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/assets/index.css]: add semicolon

File: the-telephone-app-fe/src/components/Draw/Draw.js [hunk 7]
```diff
@@ -205,7 +205,7 @@ const Draw = ({ width = "815rem", height = "350rem" }) => {
                     <img className="header-image" src={imgLogo}></img>
                     <p className="custom-font">HEY, IT'S TIME TO DRAW!</p>
                     <p className="text-center w-100 fw-bold">
-                      {dataReceive.value}
+                      {/* {dataReceive.value} */}
```
Comment (ces-manhnguyen) [inline the-telephone-app-fe/src/components/Draw/Draw.js]: uncomment this

**Review by ces-manhnguyen — APPROVED** [review]

**Pull request merged**